package.json

```diff
@@ -32,6 +32,7 @@
         "@apify/log": "^1.1.5",
         "fs-extra": "^10.0.0",
         "markdown-yaml-metadata-parser": "^3.0.0",
+        "nvm": "^0.0.4",
```

Comment:
You accidentally installed `nvm` as a dependency of the project rather than globally. Also, it's not recommended to use the NVM installed from NPM, as it's not the correct one (see [here](https://www.npmjs.com/package/nvm)). The real one lives on https://nvm.sh/

To fix this issue, run this command in your terminal within the project's directory:

```shell
npm uninstall nvm && curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.2/install.sh | bash
```

Reply:
Thanks for noticing it, Matt. I accidentally installed it locally while debugging the libjpeg error with Misha. I couldn't download packages globally because of some permission issues with my mac, but it is solved now. I will fix this in the next commit.